liquidcore/home/templates/base.html

```diff
@@ -3,6 +3,7 @@
 <title>{{ liquid_title }} - Liquid Investigations</title>
 {% load static %}
 <link rel="stylesheet" href="{% static "search/bootstrap.css" %}">
+<link rel="shortcut icon" type="image/png" href="{% static "logo_favicon/Liquid_Favicon.png" %}"> 
```

Comment:
Let's call it `icons` instead of `logo_favicon`, and no need to have the filenames begin with `Liquid_`, we already know it's liquid-core :)